packages/syft/src/syft/core/adp/data_subject_ledger.py

```diff
@@ -61,7 +61,10 @@ def convert_constants_to_indices(rdp_constant_array: np.ndarray) -> np.ndarray:
     """
     # Find indices for all RDP constants <= 50
     sub50_mask = rdp_constant_array <= 50
-    sub50_indices = (((rdp_constant_array - 1) * sub50_mask) * 10_000).astype(int)
+    # np.maximum is to avoid negative indices when rdp_constant_array is < 1
+    sub50_indices = (
+        (np.maximum(0, rdp_constant_array - 1) * sub50_mask) * 10_000
```

Comment:
MAYBE: I think the -1 should be outside like
`np.maximum(0, (rdp_constant_array  * sub50_mask * 10_000)-1)`
![Screenshot 2022-11-18 at 7 00 32 AM](https://user-images.githubusercontent.com/43314053/202595792-0736e37d-bd9b-4336-8e96-6ff8a1992acf.png)

Reply:
Great catch- fixed in the latest commit, will add a test to make sure the values are correct as well :+1: